apps/files_sharing/lib/External/Storage.php

```diff
@@ -345,14 +345,17 @@ public function isSharable($path): bool {
 
 	public function getPermissions($path): int {
 		$response = $this->propfind($path);
+		$ocsPermissions = $response['{http://open-collaboration-services.org/ns}share-permissions'];
```

Comment:
```suggestion
		$ocsPermissions = $response['{http://open-collaboration-services.org/ns}share-permissions'] ?? null;
```
Is something like that not needed? What’s the type of `$response`?

Reply:
Right, missing key already results in null but also logs an error message.